### ISSUE TITLE
plain text password in the database

### ISSUE THREAD

**Issue by ukcb** [ether/etherpad-lite#3421]
I can see my admin password in plain text in the database.

```
sessionstorage:-FWbiguBVxp1_VWLERNNheogwak9aewi {"cookie":
{"path":"/","_expires":null,"originalMaxAge":null,"httpOnly":true,"secure":false},
"passport":{},"user":{"password":"Here is a plain text password!",
"is_admin":true,"username":"admin"}}
```
That must not happen!

v1.6.6

**Comment by JohnMcLear** [comment]
Steps to replicate

1. Uncomment password section
1. Visit /admin
1. Cat var/dirty.db | grep YOURPASS

**Comment by JohnMcLear** [comment]
PR in to fix.
https://github.com/ether/etherpad-lite/pull/3782

**Comment by alasserr** [comment]
Hi, the [5c5b99f](https://github.com/anttiviljami/etherpad-lite/commit/5c5b99fc9ad33054fb0291b92084e00ae1e634ef) commit makes etherpad not usable (at least with the Docker version) :
- at first login, the admin password is the one one's set it up from Docker variables environment
- but then the login is changed to "PASSWORD_HIDDEN" 
I'm not sure it only concerns Docker versions but this is a HUGE problem.

Using etherpad/etherpad (1.8.4) with postgresQL DB. From Docker. But I tried modifying the password directly in settings.json => same issue (first login OK, second > PASSWORD_HIDDEN) so I'm not sure it is 100% Docker related.

**Comment by JohnMcLear** [comment]
Weird.  I'm not sure how doesn't affect non docker deployed versions tho?

**Comment by JohnMcLear** [comment]
I can't even get the password prompt.  I changed password and I'm not re-prompted..

```
jose@server:~/develop$ cat settings.json | grep pass | grep derp
      "password": "derp",
```

What are you doing to get the re-prompt?

**Comment by muxator** [comment]
The [bug reported by @alasser](https://github.com/ether/etherpad-lite/issues/3421#issuecomment-629282828) is **confirmed** and it's **not** related to Docker.
I should have been more thorough in https://github.com/ether/etherpad-lite/pull/3782#issuecomment-605730825, sorry.

Let's move the discussion on #4016.

**Comment by rhansen** [comment]
This issue should be repoened (the fix that closed this issue was reverted).

**Comment by JohnMcLear** [comment]
@rhansen can you think of a way to solve this issue?  It's one of the most critical for 1.9

**Comment by rhansen** [comment]
@JohnMcLear Wasn't this fixed by #4178?

**Comment by rhansen** [comment]
Oh, this is different.

Hmm... I think we can store a deep copy of the `settings.users[username]` object as `req.session.user` and remove the password field from that copy. I'll toss together a PR.
